### ISSUE TITLE
Issues with react-helmet

### ISSUE THREAD

**Issue by t1mmen** [markmichon/gatsby-plugin-feed-generator#4]
When `gasby develop`'ing with this plugin, I'm seeing
```
Warning: Each child in an array or iterator should have a unique "key" prop.

Check the top-level render call using <head>. See https://fb.me/react-warning-keys for more information.
    in link
    in HTML
```
... in the terminal. A possible workaround might be to add `data-react-helmet="true"` to the `<link>` tags generated in [gatsby-ssr.js](https://github.com/markmichon/gatsby-plugin-feed-generator/blob/master/src/gatsby-ssr.js)

**Comment by markmichon** [comment]
Thanks for the heads up @t1mmen! I'll have some time later in the week to put together a fix. Looks like a change at some point with Gatsby or Helmet that prefers more explicit helmet components passed in or, as the error notes, a need for keys. 

If you need a quicker turnaround, I'm happy to accept a PR with the fix 👍. The easy way would be to do what gatsby-plugin-feed does and drop index based keys onto the two links, at least until future versions offer more complex feed splitting.

**Comment by markmichon** [comment]
Fixed and updated on latest(v1.0.6) available on npm. Let me know if you still experience issues!
